instagram/src/components/SearchBar/Search.js

```diff
@@ -14,7 +14,7 @@ const Search = props => {
       <img alt="instagram logo" src={iglogo} className="logo-image"/>
       </div>
       <div>
-       <input className="search-bar" type="text" placeholder="Search" onInput={props.searchPosts}/>
+       <input className="search-bar" type="text" placeholder="Search" onKeyDown={props.searchPosts}/>
```

Comment:
onChange vs onKeyDown might be a better user experience here.  but onKeyDown does work.